contrib/cirrus/runner.sh

```diff
@@ -35,6 +35,11 @@ function _run_automation() {
 }
 
 function _run_validate() {
+    # FIXME: no Python.h in aarch64 image
+    if [[ $(uname -m) == "aarch64" ]]; then
+        bigto ooe.sh dnf install -y python3-devel
+    fi
```

Comment:
This is odd, is there a difference here in our image build-scripts for aarch64?

Reply:
might have been a case of me earlier using the wrong image, let me get rid of that in the next push. I need to skip some other aarch64 tests I think